flow-typed/Collections.js

```diff
@@ -14,14 +14,21 @@ declare type CollectionState = {
   pending: CollectionGroup,
   edited: CollectionGroup,
   builtin: CollectionGroup,
-  saved: Array<string>,
   isResolvingCollectionById: { [string]: boolean },
   error?: string | null,
```

Comment:
Hovering shouldn't be the only way to add queue as it doesn't work in some platforms.
Non-blocker as I guess, can think of alternatives later if don't want to expand the content menu any further.

https://odysee-workspace.slack.com/archives/C02G20Z2AEL/p1654663481645619?thread_ts=1654624401.430989&cid=C02G20Z2AEL